src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -598,6 +598,30 @@ private void DisableOngoingTasksIfNeeded(DatabaseRecord databaseRecord)
                     task.Disabled = true;
                 }
             }
+
+            if (databaseRecord.ExternalReplications != null)
+            {
+                foreach (var task in databaseRecord.ExternalReplications)
+                {
+                    task.Disabled = true;
+                }
+            }
+
+            if (databaseRecord.HubPullReplications != null)
+            {
+                foreach (var task in databaseRecord.HubPullReplications)
+                {
+                    task.Disabled = true;
```

Comment:
What if  the value was already disabled at backup time?

Reply:
It will stay disabled